File: Programmers/Level1/이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    index = 0
    s = list(s)
    
    for i in range(len(s)):
        if s[i] == " ":
            index = 0
            continue
        if index % 2 == 0:
            s[i] = s[i].upper()
            index += 1
        elif index % 2 == 1:
            s[i] = s[i].lower()
            index += 1
            
    return "".join(s)

File: Programmers/Level1/이상한 문자 만들기/이상한 문자 만들기.go
import "unicode"

func solution(s string) string {
    str := []rune(s)
    var index int
    for i:=0; i<len(str); i++{
        if unicode.IsSpace(str[i]) {
            index = 0
        }else if index%2==0{
            str[i] = unicode.ToUpper(str[i])
            index += 1
        }else{
            str[i] = unicode.ToLower(str[i])
            index += 1
        }
    }
    return string(str)
}
